### DISCUSSION TITLE
v7.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by RDIL** [thepeacockproject/Peacock#385] [Announcements]
## What's Changed

* Update locale.json by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/277
* Update locale.json by @Wuzmart in https://github.com/thepeacockproject/Peacock/pull/278
* Update locale.json by @Wuzmart in https://github.com/thepeacockproject/Peacock/pull/279
* Fix two Miami escalations not counting towards escalation challenges by @moonysolari in https://github.com/thepeacockproject/Peacock/pull/288
* Add SMF contract support (#293) by @AnthonyFuller in https://github.com/thepeacockproject/Peacock/pull/294
* [perf] Inventory rework (part 1) by @RDIL in https://github.com/thepeacockproject/Peacock/pull/291
* Bump semver from 7.5.0 to 7.5.3 by @dependabot in https://github.com/thepeacockproject/Peacock/pull/297
* hotfix: fix call to getUnlockableById in planning by @jorys-paulin in https://github.com/thepeacockproject/Peacock/pull/299
* Added french translations by @jorys-paulin in https://github.com/thepeacockproject/Peacock/pull/302
* Feature/escalations by @jorys-paulin in https://github.com/thepeacockproject/Peacock/pull/306
* Added Dashes to Escalation names to be on par with ioi missions. by @Cookie303728 in https://github.com/thepeacockproject/Peacock/pull/307
* Multi-Version Mastery and Sniper Scoring by @AnthonyFuller in https://github.com/thepeacockproject/Peacock/pull/270
* Translated escalations text by @Cookie303728 in https://github.com/thepeacockproject/Peacock/pull/311
* Translate Escalations to french by @jorys-paulin in https://github.com/thepeacockproject/Peacock/pull/312
* Update locale.json by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/313
* Update (parts of) some contracts to match newest IOI version by @grappigegovert in https://github.com/thepeacockproject/Peacock/pull/295
* Update locale.json by @Wuzmart in https://github.com/thepeacockproject/Peacock/pull/315
* 3.160 update by @riisikumi in https://github.com/thepeacockproject/Peacock/pull/316
* Update Copyright year by @million1156 in https://github.com/thepeacockproject/Peacock/pull/317
* Fix to extension filtering in GetProfile request by @vitphire in https://github.com/thepeacockproject/Peacock/pull/319
* feat(menu): add escalation tile to H2/3 dashboard by @AnthonyFuller in https://github.com/thepeacockproject/Peacock/pull/314
* Add descriptions to Peacock escalations by @jorys-paulin in https://github.com/thepeacockproject/Peacock/pull/320
* Bump actions/checkout from 3 to 4 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/321
* Update locale.json by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/322
* Fix SMF check by @prolium in https://github.com/thepeacockproject/Peacock/pull/324
* Bump get-func-name from 2.0.0 to 2.0.2 by @dependabot in https://github.com/thepeacockproject/Peacock/pull/323
* feat(escalation): pontus by @AnthonyFuller in https://github.com/thepeacockproject/Peacock/pull/328
* Update locale.json by @Amdr3B4st0s in https://github.com/thepeacockproject/Peacock/pull/329
* Update locale.json by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/330
* Bump actions/setup-node from 3 to 4 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/332
* Update locale.json by @Wuzmart in https://github.com/thepeacockproject/Peacock/pull/334
* Refactor SMF support into it's own helper class by @RDIL in https://github.com/thepeacockproject/Peacock/pull/335
* Added Spanish translation by @nachito33 in https://github.com/thepeacockproject/Peacock/pull/338
* Allow the patcher project to open and build on Visual Studio 2022 by @prolium in https://github.com/thepeacockproject/Peacock/pull/281
* Bump actions/upload-artifact from 2 to 3 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/343
* Bump actions/checkout from 3 to 4 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/341
* Bump microsoft/setup-msbuild from 1.1 to 1.3 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/342
* Change the image path in the escalation files to adapt to the changed image pack by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/347
* Update gamechanger tile image by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/348
* Fix some strict type errors by @RDIL in https://github.com/thepeacockproject/Peacock/pull/344
* Update Node.js to v20 by @RDIL in https://github.com/thepeacockproject/Peacock/pull/345
* Update locale.json by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/350
* Replace developmentAllowRuntimeRestart flag by @RDIL in https://github.com/thepeacockproject/Peacock/pull/349
* Bump github/codeql-action from 2 to 3 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/356
* Bump actions/upload-artifact from 3 to 4 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/357
* Fix contract creation not using appropriate bricks by @RDIL in https://github.com/thepeacockproject/Peacock/pull/355
* Fix peacock escalations' incomplete displayed killing tips by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/358
* Check for small item sizes for hidden stash by @solderq35 in https://github.com/thepeacockproject/Peacock/pull/359
* Separate routing logic and business logic by @RDIL in https://github.com/thepeacockproject/Peacock/pull/352
* New non-pack challenges by @moonysolari in https://github.com/thepeacockproject/Peacock/pull/363
* Bump vite from 5.0.2 to 5.0.12 by @dependabot in https://github.com/thepeacockproject/Peacock/pull/365
* Bump follow-redirects from 1.15.3 to 1.15.5 by @dependabot in https://github.com/thepeacockproject/Peacock/pull/366
* Escalation QoL fix: Shangrila & Xmas by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/370
* Fix unwanted handler voices in The CurryMaker Chaos by @moonysolari in https://github.com/thepeacockproject/Peacock/pull/374
* Remove redundant sloth challenge by @moonysolari in https://github.com/thepeacockproject/Peacock/pull/375
* Fix ArcadePageTemplate by @grappigegovert in https://github.com/thepeacockproject/Peacock/pull/373
* Bump robinraju/release-downloader from 1.8 to 1.9 in /.github/workflows by @dependabot in https://github.com/thepeacockproject/Peacock/pull/377
* Fix SMF adding contracts to a location that has no contracts yet (v7 ver) by @grappigegovert in https://github.com/thepeacockproject/Peacock/pull/380
* Revert hard-coded server version  by @moonysolari in https://github.com/thepeacockproject/Peacock/pull/381
* Fix the Christmas Calamity feat tile image by @suanjiansalt in https://github.com/thepeacockproject/Peacock/pull/383
* Fix some challenges not being ticked on mission end page by @moonysolari in https://github.com/thepeacockproject/Peacock/pull/384

## New Contributors
* @jorys-paulin made their first contribution in https://github.com/thepeacockproject/Peacock/pull/299
* @million1156 made their first contribution in https://github.com/thepeacockproject/Peacock/pull/317
* @vitphire made their first contribution in https://github.com/thepeacockproject/Peacock/pull/319
* @prolium made their first contribution in https://github.com/thepeacockproject/Peacock/pull/324
* @nachito33 made their first contribution in https://github.com/thepeacockproject/Peacock/pull/338
* @solderq35 made their first contribution in https://github.com/thepeacockproject/Peacock/pull/359

**Full Changelog**: https://github.com/thepeacockproject/Peacock/compare/v6.4.0...v7.0.0-alpha.1

<hr /><em>This discussion was created from the release <a href='https://github.com/thepeacockproject/Peacock/releases/tag/v7.0.0-alpha.1'>v7.0.0-alpha.1</a>.</em>

**Comment by PatrickJnr** [comment]
Fresh download, getting 

`TypeError: true.toLowerCase is not a function
    at Object.<anonymous> (D:\Peacock\chunk0.js:1:7623988)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49`

**Reply by RDIL** [reply]
Good catch! just published a hotfix for it!
